Tests/Imagine/Filter/Loader/RotateFilterLoaderTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Liip\ImagineBundle\Tests\Filter;
+
+use Liip\ImagineBundle\Imagine\Filter\Loader\RotateFilterLoader;
+use Liip\ImagineBundle\Tests\AbstractTest;
+
+/**
+ * @covers Liip\ImagineBundle\Imagine\Filter\Loader\RotateFilterLoader
+ */
+class RotateFilterLoaderTest extends AbstractTest
+{
+    public function testLoad()
+    {
+        $loader = new RotateFilterLoader();
+
+        $image = $this->getMockImage();
+
+        $result = $loader->load($image, array('angle' => 0));
+        $this->assertTrue($image === $result);
+
+        $image = $this->getMockImage();
+        $image
+            ->expects($this->once())
+            ->method('rotate')
+            ->with(90)
+            ->willReturn($image)
+        ;
+
+        $result = $loader->load($image, array('angle' => 90));
+        $this->assertInstanceOf('Imagine\Image\ManipulatorInterface', $result);
+    }
```

Comment:
it would also good to add a functional test for the filter service, like this one: https://github.com/liip/LiipImagineBundle/blob/master/Tests/Functional/Imagine/Filter/Loader/InterlaceFilterLoaderTest.php#L9